torch/csrc/jit/script/parser.h

```diff
@@ -64,6 +64,8 @@ struct Parser {
       std::vector<Expr> exprs = { prefix };
       while(L.cur().kind != end) {
         L.expect(',');
+        if (L.cur().kind == end)
+          break;
```

Comment:
I'm not sure I'm understanding. Do you mean:
`      L.expect(',');
      while(L.cur().kind != end) {
        exprs.push_back(parseExp());
        L.expect(',');
      }
`
This would require a trailing comma.